README.md

```diff
@@ -1,84 +1,29 @@
-- 📃 Documentation structure using [mkdocs](http://www.mkdocs.org)
-- 💬 Auto generation of change log using **gitchangelog** to keep a HISTORY.md file automatically based on your commit history on every release.
-- 🐋 A simple [Containerfile](Containerfile) to build a container image for your project.  
-  `Containerfile` is a more open standard for building container images than Dockerfile, you can use buildah or docker with this file.
-- 🧪 Testing structure using [pytest](https://docs.pytest.org/en/latest/)
-- ✅ Code linting using [flake8](https://flake8.pycqa.org/en/latest/)
-- 📊 Code coverage reports using [codecov](https://about.codecov.io/sign-up/)
-- 🛳️ Automatic release to [PyPI](https://pypi.org) using [twine](https://twine.readthedocs.io/en/latest/) and github actions.
-- 🎯 Entry points to execute your program using `python -m <wikipedia_name_query>` or `$ wikipedia_name_query` with basic CLI argument parsing.
-- 🔄 Continuous integration using [Github Actions](.github/workflows/) with jobs to lint, test and release your project on Linux, Mac and Windows environments.
-
-> Curious about architectural decisions on this template? read [ABOUT_THIS_TEMPLATE.md](ABOUT_THIS_TEMPLATE.md)  
-> If you want to contribute to this template please open an [issue](https://github.com/rochacbruno/python-project-template/issues) or fork and send a PULL REQUEST.
-
-[❤️ Sponsor this project](https://github.com/sponsors/rochacbruno/)
-
-<!--  DELETE THE LINES ABOVE THIS AND WRITE YOUR PROJECT README BELOW -->
-
 ---
 # wikipedia_name_query
 
-[![codecov](https://codecov.io/gh/M-A-T-T-Y-D/Wikipedia-Name-Query/branch/main/graph/badge.svg?token=Wikipedia-Name-Query_token_here)](https://codecov.io/gh/M-A-T-T-Y-D/Wikipedia-Name-Query)
-[![CI](https://github.com/M-A-T-T-Y-D/Wikipedia-Name-Query/actions/workflows/main.yml/badge.svg)](https://github.com/M-A-T-T-Y-D/Wikipedia-Name-Query/actions/workflows/main.yml)
 
 Awesome wikipedia_name_query created by M-A-T-T-Y-D
 
-## Install it from PyPI
-
-```bash
-pip install wikipedia_name_query
-```
+<p>This Module was created to query Wikipedia to find The Date of birth, the date of death the age and the full name of a person.<br>
+This Module utilises the terminal to input commands instead of running the code through an IDE.<br>
+Commands:<br>
+python Command.py -h = Displays every command with a description<br>
+python Commands.py age --Name "Insert Name" = this command retrieves the age of the specified person<br>
+python Commands.py name --Name "Insert Name" = This command retrieves the full name of a specified person<br>
+python Commands.py DOB --Name "Insert Name" = This command retrieves the Date of birth of a specified person<br>
+python Commands.py DOD --Name "Insert Name" = This command retrieves the date of death of a specified person<br>
+python Commands.py Output --Name "Insert Name" = This command outputs everything collected about that person</p>
```

Comment:
Similar to above, but you could rearrange each line to:

"""
To get the age of the person:
```bash
python Commands.py age --Name "Insert Name"
```
"""

"""